gson/src/test/java/com/google/gson/functional/ObjectTest.java

```diff
@@ -141,9 +141,9 @@ public void testClassWithTransientFieldsDeserialization() {
 
   @Test
   public void testClassWithTransientFieldsDeserializationTransientFieldsPassedInJsonAreIgnored() {
-    String json = "{\"transientLongValue\":1,\"longValue\":[1]}";
+    String json = "{\"transientLongValue\":5,\"longValue\":[1]}";
     ClassWithTransientFields<?> target = gson.fromJson(json, ClassWithTransientFields.class);
-    assertThat(target.transientLongValue != 1).isFalse();
```

Comment:
Very odd! Seems like it was this way [from the beginning](https://github.com/google/gson/blob/18b301dfeb91ca4468e0955d4cafcb32186efe1e/gson/src/test/java/com/google/gson/functional/ObjectTest.java#L134).